native/cocos/bindings/manual/jsb_conversions_spec.cpp

```diff
@@ -144,63 +119,32 @@ enum class MathType {
     RECT,
     COLOR,
 };
-bool uintptr_t_to_seval(uintptr_t v, se::Value *ret) { // NOLINT(readability-identifier-naming)
-    ret->setDouble(v);
-    return true;
-}
 
-bool size_to_seval(size_t v, se::Value *ret) { // NOLINT(readability-identifier-naming)
-    ret->setSize(v);
-    return true;
-}
-
-bool Vec2_to_seval(const cc::Vec2 &v, se::Value *ret) { // NOLINT(readability-identifier-naming)
-    CC_ASSERT(ret != nullptr);
-    se::HandleObject obj(se::Object::createPlainObject());
-    obj->setProperty("x", se::Value(v.x));
-    obj->setProperty("y", se::Value(v.y));
-    obj->setProperty("type", se::Value(static_cast<uint32_t>(MathType::VEC2)));
-    ret->setObject(obj);
-
-    return true;
-}
-
-bool Vec3_to_seval(const cc::Vec3 &v, se::Value *ret) { // NOLINT(readability-identifier-naming)
+bool Quaternion_to_seval(const cc::Quaternion &v, se::Value *ret) { // NOLINT(readability-identifier-naming)
```

Comment:
Could Quaternion_to_seval be deprecated?

Reply:
It's a new interface introduced in 3.6